.github/workflows/rollout-module.yaml

```diff
@@ -107,20 +110,36 @@ jobs:
         env:
           CHANNEL: ${{ inputs.channel }}
         working-directory: module-manifests/modules/istio/${{ inputs.channel }}
+        shell: bash
         run: |
           echo "Creating module configuration file:"
-          cat <<EOF | tee module-config.yaml
-          name: ${FULL_MODULE_NAME}
-          channel: ${CHANNEL}
-          version: ${TAG}
-          manifest: ${MANIFEST_FILENAME}
-          defaultCR: ${DEFAULT_CR_FILENAME}
-          annotations:
-            "operator.kyma-project.io/doc-url": "${{ vars.DOC_URL }}"
-          moduleRepo: https://github.com/kyma-project/istio.git
-          moduleSecurityScanConfig: true
-          EOF
+          if "${CHANNEL}" == "experimental"; then
```

Comment:
I think we can simply avoid duplication and errors when this needs to be updated in the future by writing it like that:
```shell
if "${CHANNEL}" == "experimental"; then
  MODULE_VERSION="${TAG}-experimental"
else
  MODULE_VERSION="${TAG}"
fi

cat <<EOF | tee module-config.yaml
  name: ${FULL_MODULE_NAME}
  channel: ${CHANNEL}
  version: ${MODULE_VERSION}
  manifest: ${MANIFEST_FILENAME}
  defaultCR: ${DEFAULT_CR_FILENAME}
  annotations:
    "operator.kyma-project.io/doc-url": "${DOC_URL}"
  moduleRepo: https://github.com/kyma-project/istio.git
  moduleSecurityScanConfig: true
EOF
```